### PR TITLE
Fix various forking issues

### PR DESCRIPTION
**Pull request by nuevoalex**
Address various forking issues:

- Finalized markets (including the forking market) cannot be migrated or have crowdsourcers disavowed
- Fork finalization can only be done once
- Actually compare REP migration totals as intended so the correct Universe wins the fork migration contest

### CODE REVIEW

File: tests/reporting/test_reporting.py [hunk 1]
```diff
@@ -259,6 +259,35 @@ def test_forking(finalizeByMigration, manuallyDisavow, localFixture, universe, m
...
+def test_finalized_fork_migration(localFixture, universe, market, categoricalMarket):
+    # Make the categorical market finalized
+    proceedToNextRound(localFixture, categoricalMarket)
+    feeWindow = localFixture.applySignature('FeeWindow', categoricalMarket.getFeeWindow())
+
+    # Time marches on and the market can be finalized
+    localFixture.contracts["Time"].setTimestamp(feeWindow.getEndTime() + 1)
+    assert categoricalMarket.finalize()
+
+    # Proceed to Forking for the binary market and finzlie it
```
Comment (adrake33) [inline tests/reporting/test_reporting.py]: Minor typo: "finzlie".

File: tests/reporting/test_reporting.py [hunk 2]
```diff
@@ -259,6 +259,35 @@ def test_forking(finalizeByMigration, manuallyDisavow, localFixture, universe, m
...
+    proceedToFork(localFixture, market, universe)
+    finalizeFork(localFixture, market, universe)
+
+    # The categorical market is finzlied and cannot be migrated to the new universe
```
Comment (adrake33) [inline tests/reporting/test_reporting.py]: Same here.

File: tests/reporting/test_reporting.py [hunk 3]
```diff
@@ -259,6 +259,35 @@ def test_forking(finalizeByMigration, manuallyDisavow, localFixture, universe, m
...
+
+    # Time marches on and the market can be finalized
+    localFixture.contracts["Time"].setTimestamp(feeWindow.getEndTime() + 1)
+    assert categoricalMarket.finalize()
+
+    # Proceed to Forking for the binary market and finalize it
+    proceedToFork(localFixture, market, universe)
+    finalizeFork(localFixture, market, universe)
+
+    # The categorical market is finazlied and cannot be migrated to the new universe
```
Comment (adrake33) [inline tests/reporting/test_reporting.py]: Still has a typo: "finazlied". ;p

Comment (nuevoalex) [reply tests/reporting/test_reporting.py]: This time for sure

**Pull request merged**